packages/app/package.json

```diff
@@ -13,10 +13,22 @@
   "dependencies": {
     "@apollo/client": "^3.0.0-beta.24",
     "@apollo/link-ws": "^2.0.0-beta.3",
+    "@react-native-community/masked-view": "^0.1.6",
+    "@react-navigation/bottom-tabs": "^5.0.0",
+    "@react-navigation/native": "^5.0.0",
+    "@react-navigation/stack": "^5.0.0",
     "graphql": "^14.5.8",
+    "install": "^0.13.0",
```

Comment:
Whats going on here? :)

Reply:
Do we need "npm" and "install" as dependencies or is this a mistake?

Reply:
def a mistake. Will fix!